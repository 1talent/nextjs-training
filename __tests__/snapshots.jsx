import Home from '../pages/index'
import { render } from '@testing-library/react'

it('renders homepage unchanged', () => {
  const { container } = render(<Home products={[]} />)
  expect(container).toMatchSnapshot()
})
