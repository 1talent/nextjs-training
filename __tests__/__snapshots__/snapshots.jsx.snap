// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <div
      class=""
    >
      <h1
        class="leading-relaxed font-primary font-extrabold text-4xl text-center text-palette-primary mt-4 py-2 sm:py-4"
      >
        Get Doggy Stickers!
      </h1>
      <p
        class="max-w-xl text-center px-2 mx-auto text-base text-gray-600"
      >
        Times are tough. Liven up your home with some cute Doggy Stickers. 🐶
      </p>
    </div>
    <div
      class="py-12 max-w-6xl mx-auto grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-x-4 gap-y-8"
    />
  </div>
</div>
`;
